hw/ip/otbn/dv/model/otbn_model.cc

```diff
@@ -542,9 +555,20 @@ static void set_err_bits(svBitVecVal *dst, uint32_t src) {
   }
 }
 
+// Extract 256-bit RND EDN data from a 4 state logic value. RND data is placed
+// into 8 element uint32_t array dst.
+static void set_rnd_data(uint32_t dst[8], svLogicVecVal *src) {
+  for (int i = 0; i < 8; ++i) {
```

Comment:
std::memcpy?

Reply:
the `svLogicVecVal` has more than one member (specifically `bval` which adds the extra bits to give you four states per bit). We only care about `aval` hence extracting it here, this isn't a straight copy.

Reply:
Could you add a check to make sure the value is known here? We don't handle `x`'s in the ISS, so it's probably best to explode if some creep in.